src/utils/liveTickerTooltip.js

```diff
@@ -0,0 +1,145 @@
+// This utility script is used to show a tooltip with the full name of 
+// an equity when hovering over its ticker symbol.
+
+(()=>{
+  const API_URL = 'https://stagehand-api.composer.trade/api/v1/public/quotes';
```

Comment:
on a side not here I want to organize the files a little better. I would like to make a file that only interacts with the api. And then use the chrome extension file import system.

Just haven't taken the time to implement this. I also don't know how it will work across worlds.
https://medium.com/@otiai10/how-to-use-es6-import-with-chrome-extension-bd5217b9c978

Reply:
Yah, that would be great.